### PR TITLE
Effigy overhaul

### PR DESCRIPTION
**Pull request by Petally**
## About the PR
basic effigy overhaul (no fire just yet)

## Why / Balance
cuz i thought it would be funny i think

## Technical details
moved all relevant files under funkystation folders

**Changelog**
:cl:
- add: added breathing sounds to effigy
- add: effigy can now be destroyed and turns into gibs when so
- add: effigy now gets hurt when hitting walls

### CODE REVIEW

**Comment by Petally** [comment]
i am using this effigy to learn codebase. very cool very swag I like it.

File: Resources/Prototypes/_Funkystation/Entities/Objects/Fun/the_effigy.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                min: 1
+                max: 1
+              OrganHumanBrain:
+                min: 1
+                max: 1
+      - !type:SpillBehavior
+        solution: drink
+      - !type:PopupBehavior
+        popup: effigy-destroyed
+        type: Local
```
Comment (taydeo) [inline Resources/Prototypes/_Funkystation/Entities/Objects/Fun/the_effigy.yml:67]: `type` to `popupType`

**Review by taydeo — APPROVED** [review]

File: Resources/Prototypes/_Funkystation/Entities/Objects/Fun/the_effigy.yml (hunk 1 above)
Comment (taydeo) [reply Resources/Prototypes/_Funkystation/Entities/Objects/Fun/the_effigy.yml:67]: didnt mean to approve, change this pls

Comment (Petally) [reply Resources/Prototypes/_Funkystation/Entities/Objects/Fun/the_effigy.yml:67]: ok

**Comment by Petally** [comment]
fixed

**Pull request merged**